src/views/Login.vue

```diff
@@ -92,9 +94,15 @@ export default class Login extends Vue {
           // Don't redirect automatically; we handle that above.
           return false;
         },
+        uiShown: () => this.recordEvent('loginShown'),
       },
     });
   }
+
+  @Watch('ready')
+  onReady(val: boolean) {
+    if (val) this.logReady('login_loaded');
```

Comment:
Same comment for 'login_loaded". Also do we have different naming conventions for loginShown vs login_loaded?

Reply:
i'm using hacker_style for log codes since i find it more readable, and camelCase for properties in log record payloads since it's JSON and that's the convention for JSON.